example_recursive_test.go

```diff
@@ -24,14 +26,18 @@ func recursiveServeHTTP(l net.Listener) {
 	s := &http.Server{
 		Handler: &recursiveHTTPHandler{},
 	}
-	s.Serve(l)
+	go func() {
+		if err := s.Serve(l); err != cmux.ErrListenerClosed {
+			log.Panic(err)
+		}
+	}()
 }
 
 func tlsListener(l net.Listener) net.Listener {
 	// Load certificates.
 	certificate, err := tls.LoadX509KeyPair("cert.pem", "key.pem")
 	if err != nil {
-		log.Fatal(err)
+		log.Panic(err)
```

Comment:
OK.

On Thu, Jan 7, 2016 at 12:02 AM, Soheil Hassas Yeganeh <
notifications@github.com> wrote:

> In example_recursive_test.go
> <https://github.com/soheilhy/cmux/pull/5#discussion_r49042304>:
>
> >  }
> >
> >  func tlsListener(l net.Listener) net.Listener {
> >  	// Load certificates.
> >  	certificate, err := tls.LoadX509KeyPair("cert.pem", "key.pem")
> >  	if err != nil {
> > -		log.Fatal(err)
> > +		log.Panic(err)
>
> Sure, they are different, but since this is just an example, perhaps a
> simple panic will do just fine?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/soheilhy/cmux/pull/5/files#r49042304>.
>